From 4984c94679f5fd4362cb31a3b557d420a83f1962 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 16 Oct 2014 16:01:54 -0400
Subject: [scsi] mpt2sas: Bump mpt2sas driver version to 17.100.00.00

Message-id: <1413475335-14737-12-git-send-email-thenzl@redhat.com>
Patchwork-id: 97596
O-Subject: [RHEL7.1 e-stor PATCH 11/32] mpt2sas: Bump mpt2sas driver version to 17.100.00.00
Bugzilla: 1088524
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

Bump mpt2sas driver version to 17.100.00.00

Signed-off-by: Sreekanth Reddy <Sreekanth.Reddy@avagotech.com>
Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 49563e1e4b202436a36409f365b0a96796db5aad)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/mpt2sas/mpt2sas_base.h b/drivers/scsi/mpt2sas/mpt2sas_base.h
index 6ce7020..e0e4dd4 100644
--- a/drivers/scsi/mpt2sas/mpt2sas_base.h
+++ b/drivers/scsi/mpt2sas/mpt2sas_base.h
@@ -69,8 +69,8 @@
 #define MPT2SAS_DRIVER_NAME  "mpt2sas"
 #define MPT2SAS_AUTHOR "LSI Corporation <DL-MPTFusionLinux@lsi.com>"
 #define MPT2SAS_DESCRIPTION "LSI MPT Fusion SAS 2.0 Device Driver"
-#define MPT2SAS_DRIVER_VERSION  "16.100.00.00"
-#define MPT2SAS_MAJOR_VERSION  16
+#define MPT2SAS_DRIVER_VERSION  "17.100.00.00"
+#define MPT2SAS_MAJOR_VERSION  17
 #define MPT2SAS_MINOR_VERSION  100
 #define MPT2SAS_BUILD_VERSION  00
 #define MPT2SAS_RELEASE_VERSION  00
-- 
1.7.1